runtime/service/src/main/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalog.java

```diff
@@ -508,6 +508,11 @@ private void createNamespaceInternal(
     } else {
       LOGGER.debug("Skipping location overlap validation for namespace '{}'", namespace);
     }
+    if (!realmConfig.getConfig(
+        FeatureConfiguration.ALLOW_NAMESPACE_CUSTOM_LOCATION, catalogEntity)) {
+      LOGGER.debug("Validating that namespace {} has a location inside its parent", namespace);
```

Comment:
Minor: is it necessary to have another debugging message here given line 1106 provides a similar one?